docs/one-wire-devices.md

```diff
@@ -0,0 +1,150 @@
+
+**WARNING:** Some One-Wire devices will require a pull-up resistor on the Data line (DQ). For example, the popular DS18B20 temperature sensor requires a 4.7 kΩ pull-up resistor on the DQ to operate properly.
+
+### Stage 2: Connect Power & Ground
+
+Connect the power and ground wires belonging to your One-Wire device to the corresponding power and ground pins on the Omega2 board. 
+
+### Stage 3: Connect the DQ
+
+Connect the DQ from your One-Wire device to the GPIO19 pin on the Omega2 board. 
+
+<h2 id="enable">Enable One-Wire Device Support</h2>
+
+
+Once the physical connections are made, install dts overlay package to register a one-wire master in Linux associated with GPIO19 to communicate with the one-wire slave device.  
+
+**Step 1:**  Install the `onion-dt-overlay-w1-gpio` package using opkg
+
+<pre>
+<code>opkg update
+opkg install onion-dt-overlay-w1-gpio</code>
+</pre>
+
+**Step 2:** After the package is successfully installed, GPIO19 will act as a One-Wire Master. 
+
+<pre>
+<code>/sys/devices/w1_bus_master1</code>
+</pre>
+
+<h2 id="find">Find Connected One-Wire Devices</h2>
+
+**Step 1:** Navigate to the relevant directory where One-Wire devices are listed. This is under `/sys/devices/w1_bus_master1`
+
+<pre>
+<code>cd /sys/devices/w1_bus_master1
+ls</code>
+</pre>
+
+**Step 2:** Perform a check to find if slave devices are connected or not. 
+
+<pre>
+<code>cat /sys/devices/w1_bus_master1/w1_master_slave_count</code>
+</pre>
+
+**Step 3:** The command in step 2 will return the count of the number of slave devices connected. 
+
+| Result            | Interpretation                            |
+|-------------------|-------------------------------------------|
+| 1                 | A slave device is connected               |
+| 0                 | A slave device is connected               |
+
+**NOTE:** The One-Wire bus master kernel module scans the data pin every 10 seconds for new devices connected, if any, hence, it is recommended to wait a little while and try again. 
+
+**Step 4:** If a slave device is connected, run the following command: 
+
+<pre>
+<code>cat/sys/devices/w1_bus_master1/w1_master_slaves </code>
+</pre>
+
+This will return a (newline delimited) list of your slave device ID(s). The Device ID is the serial number of your One-Wire device. Example of a device ID: 28-000123456789. 
+
+**NOTE:** Each device will have a different serial number, making it complex to use One-Wire devices programmatically. 
+
+<h2 id="read">Read Data from a Connected One-Wire Device</h2>
+
+**Step 1:** Navigate to the specific slave device directory and access the data from the __w1_slave file__ by running the following command: 
+
+<pre>
+<code>cat /sys/devices/w1_bus_master1/DeviceID/w1_slave</code>
+</pre>
+
+The `<DeviceID>` in the aforesaid command is the serial number of your One-Wire device. Please refer to **Step 4** of the aforesaid section "Find Connected One-Wire Devices" to learn more about finding the "DeviceID". 
```

Comment:
I know this is a nitpick, but I find the word aforesaid awkward. Can you please change it to something else?